universal-login-wallet/src/core/services/ConfigService.ts

```diff
@@ -11,7 +11,7 @@ export class ConfigService {
 
   async fetchRelayerConfig() {
```

Comment:
Is that service necessary?  We have this functionality in sdk already?

Reply:
Agree, let's delete it in next PR